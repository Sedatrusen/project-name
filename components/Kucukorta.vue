<template>
  <div class="sayfa">
    <div class="aralık">
      <div class="blizardgames"><p class="aralık-yazı">BLIZZARD GAMES</p></div>
      <div class="simgeler">
        <nuxt-link to="product"
          ><img class="aralık-simge" src="../assets/aralık1.svg"
        /></nuxt-link>

        <img class="aralık-simge" src="../assets/aralık2.svg" />
        <img class="aralık-simge" src="../assets/aralık3.svg" />
        <img class="aralık-simge" src="../assets/aralık4.svg" />
        <img class="aralık-simge" src="../assets/aralık5.svg" />
        <img class="aralık-simge" src="../assets/aralık6.svg" />
        <img class="aralık-simge" src="../assets/aralık7.svg" />
        <img class="aralık-simge" src="../assets/aralık8.svg" />
        <img class="aralık-simge" src="../assets/aralık9.svg" />
      </div>
      <div>
        <div class="partner">
          <div class="blizardgames">
            <p class="aralık-yazı">PARTNER GAMES</p>
          </div>
          <img class="aralık-simge" src="../assets/kucukortaprt1.svg" />
          <img class="aralık-simge" src="../assets/kucukortaprt2.svg" />
          <img class="aralık-simge" src="../assets/kucukortaprt3.svg" />
          <img class="aralık-simge" src="../assets/kucukortaprt4.svg" />
        </div>
      </div>
    </div>
  </div>
</template>

<script>
export default {}
</script>

<style scoped>
.sayfa {
  background-image: url('../assets/top.png');
  background-color: #01234a;
}

.aralık {
  width: 100%;
  height: 60px;
  background-color: transparent;
  border-bottom: 1px solid rgba(255, 255, 255, 0.15);
  border-top: 1px solid rgba(255, 255, 255, 0.15);
  display: flex;
  align-items: center;
  justify-content: center;
}
.aralık-yazı {
  font-size: 14px;
  line-height: 1.3;
  color: white;
}
.aralık-simge {
  height: 40px;
  width: 40px;
  margin-left: 10px;
}
.aralık-simge:hover {
  transform: scale(1.1);
}
.partner {
  margin: 20px;
  padding-left: 20px;
  display: flex;
  border-left: 1px solid rgba(255, 255, 255, 0.3);
}
</style>
