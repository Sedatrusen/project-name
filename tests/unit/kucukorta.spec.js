import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vuex from 'vuex'
import Kucukorta from '@/components/Kucukorta.vue'

const localVue = createLocalVue()

localVue.use(Vuex)

describe('Kucukorta component', () => {
  let state
  let store

  beforeEach(() => {
    state = {
      carouselitems: jest.fn(),
    }
    store = new Vuex.Store({
      state,
    })
  })

  it('Kucukorta resimleri set etme', () => {
    const wrapper = shallowMount(Kucukorta, {
      mocks: { $route: { params: { id: 1 } } },
      computed: {
        blizzardgames: () => 'value_2',
        partnergames: () => '2',
      },

      localVue,
      store,
    })
    expect(wrapper.exists()).toBe(true)
    expect(wrapper).toMatchSnapshot()
  })
})
