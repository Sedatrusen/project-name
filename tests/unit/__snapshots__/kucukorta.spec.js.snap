// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kucukorta component Kucukorta resimleri set etme 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
